ERP/resources/views/job-backlog-page.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.master')
+@section('inside-body-tag')
+<div class="container-fluid">
```

Comment:
@abirshah Do you think you can add this part to a MODAL in the jobs page?

Reply:
@abirshah Looks great now man! Thanks
- However, i think you forgot to remove the old backlog button
![image](https://user-images.githubusercontent.com/31664874/110401310-54e10b00-8047-11eb-9e53-e5fb853abe8f.png)